app/views/admin/tool_rentings/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t('tool_renting.title')) %>
+<div class="col-md-8 col-md-offset-2 col-sm-12 reg-page">
+  <div class="headline">
+    <h2><%= t('tool_renting.edit') %></h2>
```

Comment:
This can now be done with `title(t('tool_renting.edit'))` and then you can remove the first `provide` row.